frontend/public/components/monitoring/kebab-dropdown.tsx

```diff
@@ -0,0 +1,22 @@
+import { Dropdown, DropdownPosition, KebabToggle } from '@patternfly/react-core';
+import * as React from 'react';
+
+import { useBoolean } from './hooks/useBoolean';
+
+const KebabDropdown: React.FC<{ dropdownItems: any[] }> = ({ dropdownItems }) => {
+  const [isOpen, setIsOpen, , setClosed] = useBoolean(false);
+
+  return (
+    <Dropdown
+      data-test-id="kebab-button"
+      dropdownItems={dropdownItems}
+      isOpen={isOpen}
+      isPlain
+      onSelect={setClosed}
+      position={DropdownPosition.right}
+      toggle={<KebabToggle id="toggle-kebab" onToggle={setIsOpen} />}
```

Comment:
This id won't be unique and will affect a11y. Maybe it can be passed as a prop if is strictly required for testing or just fallback to the `aria-label`.

Reply:
Thanks @jgbernalp and sorry for missing that!

I changed it to have an `aria-label` attribute instead of an `id`.